chain/api/Cargo.toml

```diff
@@ -18,10 +18,15 @@ async-std = { workspace = true }
 async-trait = { workspace = true }
 async-lock = { workspace = true }
 futures = { workspace = true }
+semver = "1.0"
```

Comment:
Let's move `semver` into workspace deps, because we're also using it in `core-strategy`